www/front_src/src/Resources/Graph/Performance/TimePeriods/index.tsx

```diff
@@ -1,56 +1,111 @@
 import * as React from 'react';
 
 import { useTranslation } from 'react-i18next';
-import { map, pick } from 'ramda';
+import { always, cond, lte, map, pick, T } from 'ramda';
+import { ParentSize } from '@visx/visx';
 
 import { Paper, makeStyles, ButtonGroup, Button } from '@material-ui/core';
 
-import { TimePeriodId, timePeriods } from '../../../Details/tabs/Graph/models';
+import {
+  ChangeCustomTimePeriodProps,
+  CustomTimePeriod,
+  TimePeriodId,
+  timePeriods,
+} from '../../../Details/tabs/Graph/models';
+
+import CustomTimePeriodPickers from './CustomTimePeriodPickers';
 
 const useStyles = makeStyles((theme) => ({
   header: {
-    padding: theme.spacing(2),
+    padding: theme.spacing(1, 0.5),
+    display: 'grid',
+    gridTemplateColumns: `repeat(2, auto)`,
+    columnGap: `${theme.spacing(2)}px`,
+    justifyContent: 'center',
+  },
+  buttonGroup: {
+    alignSelf: 'center',
+  },
+  button: {
+    fontSize: theme.typography.body2.fontSize,
   },
 }));
 
+const normalStep = 640;
+const largeStep = 820;
```

Comment:
We should use mui breakpoints: https://material-ui.com/customization/breakpoints/